eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -1,8 +1,14 @@
 parameters:
   Directory: 'not-specified'
   IgnoreLinksFile: "$(Build.SourcesDirectory)/eng/ignore-links.txt"
-  WorkingDirectory: $(System.DefaultWorkingDirectory)
-  ScriptDirectory: eng/common/scripts
+  WorkingDirectory: "$(System.DefaultWorkingDirectory)"
+  ScriptDirectory: "eng/common/scripts"
+  Urls: "$(dir -r -i *.md)"
+  RootUrl: "file://${{ parameters.WorkingDirectory }}/${{ parameters.Directory }}" 
```

Comment:
I don't think this is going to work. I don't believe you can asset other template parameters when initializing template parameters. It is worth a test but I suspect it will not work.

Reply:
I have already changed back. Could you check the latest commit?